crawler.js

```diff
@@ -36,7 +36,10 @@ async function openBrowser(proxyHost) {
         args.devtools = true;
     }
     if (proxyHost) {
-        args.args = [`--proxy-server=${proxyHost}`];
+        args.args = [
+            `--proxy-server=${proxyHost}`,
+            `--host-resolver-rules="MAP * ~NOTFOUND , EXCLUDE ${proxyHost.split(":")[0].replace("socks5://", "")}"`
```

Comment:
It'd be a bit more solid if we used URL parser (and as a bonus we get proxyHost validation):

```js
let url;
try {
  url = new URL(proxyHost);
} catch(e) {
  LOG('Invalid proxy URL');
}

`… EXCLUDE ${url.hostname} …`
```